samples/check-in-current-location/csharp/CheckInLocation/Models/AdaptiveCardAction.cs

```diff
@@ -0,0 +1,39 @@
+﻿// <copyright file="AdaptiveCardAction.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+
+namespace CheckInLocation.Models
+{
+    using Microsoft.Bot.Schema;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Adaptive card action model class.
+    /// </summary>
+    public class AdaptiveCardAction
```

Comment:
Why this class contains Longitude & Latitude, are these actions?

Reply:
Longitude & Latitude needs to pass fetch/async method for view location hence added in this class